DEPRECATED.md

```diff
@@ -151,7 +151,59 @@ N/A
 
 ## [opentelemetry-cpp Exporter]
 
-N/A
+### ZPages exporter
+
+#### Announcement (ZPages)
+
+* Version: 1.11.0
+* Date: 2023-09-01
+* PR: [DEPRECATION] Deprecate ZPAGES
+  [#2291](https://github.com/open-telemetry/opentelemetry-cpp/pull/2291)
+
+#### Motivation (ZPages)
+
+The ZPages specification itself was introduced in 2020,
+and has been experimental ever since,
+not getting a lot of attention, and never reaching a stable status.
+
+Several other opentelemetry projects have implemented support for zpages,
+only to later deprecate and then remove the zpages implementation (Java,
+C#), abandoning the zpages feature.
+
+In this context, it does not make sense to continue to maintain the zpages
+code in opentelemetry-cpp.
+
+#### Scope (ZPages)
+
+The following are deprecated and planned for removal:
+
+* all the API headers located under `ext/include/opentelemetry/ext/zpages/`, including:
+  * the C++ class `ThreadsafeSpanData`
+  * the C++ class `TracezDataAggregator`
+  * the C++ class `TracezHttpServer`
+  * the C++ class `TracezSpanProcessor`
+  * the C++ class `TracezSharedData`
+  * the C++ class `ZPages`
+  * the C++ class `zPagesHttpServer`
+* all the code and doc located under `ext/src/zpages/`
+* all the tests located under `ext/test/zpages/`
+* the zpages exporter library(`opentelemetry_zpages`)
+* the zpages build options in CMake (`WITH_ZPAGES`)
+
+#### Mitigation (ZPages)
+
+Consider using a different exporter,
+for example the OTLP exporter (both OTLP HTTP and OTLP GRPC are supported),
+to expose trace data to a separate trace backend.
+
+Note that this changes the access pattern:
+
+* with zpages, data is only available locally (in process)
+* with other exporters, data is available externally (not in process)
+
```

Comment:
Nit - Probably we can also mention if someone is using ZPages, they can take ownership of this component, and add it to the contrib repo -  similar to how Golang maintains [ZPages ](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/zpages)implementation.

Reply:
Thanks. Fixed.